### ISSUE TITLE
Use neater way to build the objects : YAML parameter file and object factory for example

### ISSUE THREAD

**Issue by SebastienBocquet** [SebastienBocquet/MulticopterDynamics#2]

**Comment by jgiret** [comment]
@SebastienBocquet, I see that a lot of your parameters are spread over several files. I think it would be wise to define a Configuration singleton which would read its data on the Yaml (or whatever format we want). The several modules would then be able to read their parameters from the Configuration singleton. I'll try to propose a PR soon.

@FrancoisGallard, @gomar, maybe you have an idea about that?

**Comment by FrancoisGallard** [comment]
Yep sure, the metadata should be separated from the source code.
Here you mix parameter values and logic in the same files which is an issue for tests and re-running a simulation with a different logic but the same parameters. For instance in initMeca.py:
`armLength = 0.5#<- parameters
arbitraryMass = 0.18 #<- parameters

d = {'mass': engine.getMass() + propeller.getMass() + arbitraryMass,  # <- logic
'xG': 0.,#<- parameters
'yG': 0.5 * armLength,
'zG': 0.}`

YAML is an option, python config files are another. YAML is maybe more multilanguage, config files may be more human readable.

https://wiki.python.org/moin/ConfigParserExamples
